check/check.go

```diff
@@ -216,91 +213,41 @@ func isShellCommand(s string) bool {
 	return false
 }
 
-func runAuditCommands(c BaseCheck) (out bytes.Buffer, errmsgs string, state State) {
+func runAuditCommands(c BaseCheck) (output, errMessage string, state State) {
 
 	// If check type is manual, force result to WARN.
 	if c.Type == "manual" {
-		return out, errmsgs, WARN
+		return output, errMessage, WARN
 	}
 
 	if c.Type == "skip" {
-		return out, errmsgs, INFO
+		return output, errMessage, INFO
 	}
 
 	// Check if command exists or exit with WARN.
 	for _, cmd := range c.Commands {
 		if !isShellCommand(cmd.Path) {
 			glog.V(1).Infof("%s: command not found", cmd.Path)
-			return out, errmsgs, WARN
+			return output, errMessage, WARN
 		}
 	}
 
 	// Run commands.
 	n := len(c.Commands)
 	if n == 0 {
 		// Likely a warning message.
-		return out, errmsgs, WARN
+		return output, errMessage, WARN
 	}
 
-	// Each command runs,
-	//   cmd0 out -> cmd1 in, cmd1 out -> cmd2 in ... cmdn out -> os.stdout
-	//   cmd0 err should terminate chain
-	cs := c.Commands
-
-	// Initialize command pipeline
-	cs[n-1].Stdout = &out
-	i := 1
-
-	var err error
-	errmsgs = ""
-
-	for i < n {
-		cs[i-1].Stdout, err = cs[i].StdinPipe()
-		errmsgs += handleError(
-			err,
-			fmt.Sprintf("failed to run: %s\nfailed command: %s",
-				c.Audit,
-				cs[i].Args,
-			),
-		)
-		i++
-	}
+	out, err := exec.Command("bash", "-c", c.Audit).Output()
```

Comment:
Can you change this to "sh" as I am not sure "bash" is available on all platforms